models/organization-extension/src/oidc-sso-config/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as random } from './builder';
```

Comment:
We will also need a draft model for this one ([reference](https://github.com/commercetools/merchant-center-services/blob/main/packages/settings/lib/schemas/organization-extension.graphql#L50))